ci/Jenkinsfile.android

```diff
@@ -6,7 +6,7 @@ pipeline {
   options {
     timestamps()
     /* Prevent Jenkins jobs from running forever */
-    timeout(time: 20, unit: 'MINUTES')
+    timeout(time: 30, unit: 'MINUTES')
```

Comment:
why do we need this change ?

Reply:
The build kept timing out, we discussed it with @jakubgs , we can set it to 25 minutes